src/components/Popover/Popover.stories.js

```diff
@@ -15,15 +15,35 @@
 
 /* eslint-disable react/prop-types */
 import React, { useState } from 'react';
+import { findByText, fireEvent, getByText } from '@testing-library/dom';
 
 import Button from '../Button';
 import Card from '../Card';
 
 import Popover from './Popover';
 
+const interactionTasks = [
```

Comment:
I'm wondering if this should not instead go to the spec (or elsewhere), since it uses the testing library to run interactions. At the same time it's very tied to the Storybook itself and shouldn't be run by jest.

Not the biggest fan of this interface to measure interactions, but it's very useful metrics (arguably more than simply render time)